flow/actorcompiler/actorcompiler.csproj

```diff
@@ -12,7 +12,6 @@
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <OutputPath>$(SolutionDir)bin\$(Configuration)\</OutputPath>
-    <IntermediateOutputPath>$(SystemDrive)\temp\msvcfdb\$(Configuration)\actorcompiler\</IntermediateOutputPath>
```

Comment:
Why is this being removed?  was it meaningful before ?

Reply:
It clearly doesn't work anywhere that doesn't have `C:\\temp\msvcfdb\Release\actorcompiler\` on disk...

But yaeh, my assumption is this will break your current Windows-build

Reply:
I'll go kick off an internal windows build.

Reply:
Nope, apparently it still builds fine.  No idea what it was for then.

Reply:
I'm removing it so that c sharp tooling works without that path present. It's not strictly necessary for this pull request, but I wanted language support from my IDE for c sharp before I made changes to c sharp files :)

Reply:
I don't know what this was accomplishing either. If the build works without it, then that seems ok to me.